### ISSUE TITLE
Tooltip of 'Filter Subscription' button is not accessible for keyboard only users.

### ISSUE THREAD

**Issue by sohelmulani1998** [microsoft/vscode#153070]
**GitHubTags:**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #VisualStudioCode-Win32-Apr2021; #Win32; #FTP;  #Keyboard; #WCAG2.1.1

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps:**

1. Open Visual studio code 
2. Tab till extension button and search for "Azure Functions"
3. Tab till install and and press enter now tab till the "Azure" button and press enter.
4. Tab till filter button and press enter
5. Check if tooltip is accessible to keyboard only users

**Actual:**
Tooltip of 'Filter Subscription' button is not accessible for keyboard only users.

Note: Similar issue observed for the all controls of toolbar as well

**Expected Result:**
Tooltip should be accessible to keyboard only users as well

**Attachment:**
![3120_A11y_Azure Tools for VS Code_Azure Function_Filter Subscriptions_Keyboard](https://user-images.githubusercontent.com/85861804/164215830-4d835cd4-d9ab-408c-bf00-cc6a6bc6ae56.gif)

**Comment by sohelmulani1998** [comment]
filing a new bug because old one(https://github.com/microsoft/vscode/issues/52016) was closed.

**Comment by sandy081** [comment]
This seems to be the case for all actions in VS Code

**Comment by isidorn** [comment]
This is a duplicate of https://github.com/microsoft/vscode/issues/132344
